requirements.txt

```diff
@@ -1,9 +1,31 @@
+alembic==1.3.0
```

Comment:
Instead of pinning the version of every package, we should only pin the versions of the packages we're directly using, and allow the sub-dependencies to be installed at whatever version pip determines